release/models/system/openconfig-aaa-radius.yang

```diff
@@ -26,7 +26,15 @@ submodule openconfig-aaa-radius {
     related to the RADIUS protocol for authentication,
     authorization, and accounting.";
 
-  oc-ext:openconfig-version "1.0.0";
+  oc-ext:openconfig-version "1.1.0";
+
+  revision "2025-07-04" {
+    description
+      "Add accounting-method and authorization-method leaf-lists
+      inside accounting/events/event and authorization/events/event
+      lists, respectively.";
+    reference "1.1.0";
+  }
```

Comment:
No actual change in this file (maybe vestigal from earlier drafts?). Same with the tacacs file.

Reply:
This is actually only added in the latest draft to satisfy a linter check which states submodules `openconfig-aaa-radius.yang` and `openconfig-aaa-tacacs.yang` don't share the same version as the main `openconfig-aaa.yang` module.